tests/integration/targets/s3_bucket/inventory

```diff
@@ -10,6 +10,7 @@ encryption_bucket_key
 encryption_sse
 public_access
 acl
+object_lock
```

Comment:
Another "special case".  This set of tests runs multiple tests in parallel, don't worry about this one